tests/test_message.py

```diff
@@ -35,7 +35,7 @@
 from dateutil.tz import tzutc
 
 from chatto.channel import Channel
-from chatto.message import Message
+from chatto.message import Message, MessageTypes
```

Comment:
```suggestion
from chatto.message import Message, MessageType
```
Make the changes above in this file too.